smoke-tests/designer/features/actions/forms.js

```diff
@@ -0,0 +1,59 @@
+const { formRunner } = require("../pageobjects/pages");
+const formData = require("../../data/formData");
+
+class Forms {
+  cgTest() {
```

Comment:
Similar comment to my first comment here

Reply:
@gstevenson Fixed